tests/wpt/web-platform-tests/fetch/nosniff/stylesheet.html

```diff
@@ -3,28 +3,36 @@
 <script src=/resources/testharnessreport.js></script>
 <div id=log></div>
 <script>
-  var fails = ["", "?type=", "?type=x", "?type=x/x"],
-      passes = ["?type=text/css", "?type=text/css;blah"]
+  var fails = [null, "", "x", "x/x"],
+      passes = ["text/css", "text/css;charset=utf-8"]
```

Comment:
@seanmonstar Isn't a part of the header being invalid supposed to invalidate the whole parsing? Shouldn't parsing `text/css;blah` return an error?